pdns/recursordist/docs/lua-scripting/netmask.rst

```diff
@@ -94,9 +94,12 @@ They can be matched against netmasks objects:
 
 Prefixing a mask with ``!`` excludes that mask from matching.
 
-.. function:: newNMG() -> NetMaskGroup
+.. function:: newNMG([masks]) -> NetMaskGroup
 
-  Returns a new, empty :class:`NetMaskGroup`.
+  Returns a new :class:`NetMaskGroup`.
```

Comment:
Please add a `versionchanged` that the masks are supported from 4.6 and up (as this is recursor).